.github/workflows/in_develop_labeler_preflight.yml

```diff
@@ -0,0 +1,34 @@
+name: On Develop PR Merge
+on: 
+  pull_request:
+    types: [closed]
+    branches: [develop]
+jobs:
+  save_pr_num_in_artifact:
+    name: Preflight Label Issue In Develop
+    runs-on: ubuntu-18.04
+    if: github.event.pull_request.merged == true
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Python v3.8
+        uses: actions/setup-python@v2
+        with: 
+          python-version: 3.8
+
+      - name: Install dependencies 
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r ./.github/scripts/requirements.txt
+          
```

Comment:
Maybe we can delete this part.
```suggetion
```

Reply:
Probably, will have to test it first though

Reply:
Not sure that will work. It is needed for the `icomoon_build.py` script, AFAIK.

Reply:
@Panquesito7 that sceipt isn't ran from this workflow though. At least as far as I can tell

Reply:
Whoops, right. Got confused with another script. 😅
We can remove that and it should work. You can still test if you want, though.

Reply:
Ehh what the heck. Let's just remove this part of the code. It's probably fine, and it doesn't look like I'll have time to test it anytime soon :/

I'll remove the code for python for now and keep the PR open for a bit. Feel free to test it if you want and have time :)